service/src/main/java/bio/terra/tanagra/service/authentication/JWTAccessTokenUtils.java

```diff
@@ -0,0 +1,112 @@
+package bio.terra.tanagra.service.authentication;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.exceptions.JWTVerificationException;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import com.auth0.jwt.interfaces.JWTVerifier;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PublicKey;
+import java.security.interfaces.RSAPublicKey;
+import java.security.spec.EncodedKeySpec;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+import org.bouncycastle.util.io.pem.PemObject;
+import org.bouncycastle.util.io.pem.PemReader;
+import org.springframework.util.Assert;
+
+public class JWTAccessTokenUtils {
+  private final JWTVerifier JWTVerifier;
+  private final ObjectMapper mapper = new ObjectMapper();
+  private final Base64.Decoder decoder = Base64.getUrlDecoder();
+
+  private static byte[] parsePEMFile(File pemFile) throws IOException {
+    if (!pemFile.isFile() || !pemFile.exists()) {
+      throw new FileNotFoundException(
+          "Public key pem file not found: " + pemFile.getAbsolutePath());
+    }
+    PemReader reader = new PemReader(new FileReader(pemFile));
+    PemObject pemObject = reader.readPemObject();
+    byte[] content = pemObject.getContent();
+    reader.close();
+    return content;
+  }
+
+  private static PublicKey getPublicKey(byte[] keyBytes, String algorithm) {
+    /*
+     if the file contains certificate, public key must be extracted from it
+     # cat <cert.file> | openssl x509 -pubkey -noout > pub.file
+
+     CertificateFactory cf = CertificateFactory.getInstance("X.509");
```

Comment:
Just delete this code if it's not being used. Don't the above instructions remove the need for it anyways? Or support both if you think it's needed.

Reply:
This is information is found easily online, removed. 
No need to support both